modules/host/contractcreation.go

```diff
@@ -119,14 +113,7 @@ func verifyContract(contract consensus.FileContract, terms modules.ContractTerms
 func (h *Host) acceptContract(txn consensus.Transaction) error {
 	contract := txn.FileContracts[0]
 	duration := uint64(contract.Expiration - contract.Start)
-
-	penalty := h.Collateral
-	penalty.Mul(consensus.NewCurrency64(contract.FileSize))
-	err := penalty.Mul(consensus.NewCurrency64(duration))
-	// TODO: move this check to a different function?
-	if err != nil {
-		return err
-	}
+	penalty := h.Collateral.Mul(consensus.NewCurrency64(contract.FileSize)).Mul(consensus.NewCurrency64(duration))
```

Comment:
I too prefer the former.

DurationSize could be added to info.go, but it's not ever used inside of the consensus package so that's the only place it would belong. I also think it's probably not worth it.